### ISSUE TITLE
Start button in Admin section is not working for a real user's added download link 

### ISSUE THREAD

**Issue by demon-36** [scorelab/Bassa#806]
**Describe the bug**
`Start button` in `Admin` section to start downloading files from added link is not working, when a real user (not `rand`) adds download link and requests for download the files

**To Reproduce**
1. Sign UP as real user (say user `x`)
2. Add a download link
3. log out from user `x` and login as `admin`
4. hit the start button in admin section

**Expected behavior**
After hitting `Start` button, It should start the process to download files as it does for `rand` user

**Comment by vivonk** [comment]
I think the user should be admin to start downloads @demon-36 
Can you recheck if the user `x` is admin or not. Also, if not admin, is `Admin section` visible?

**Comment by demon-36** [comment]
@vivonk  yeah you are correct user should be admin to start downloads and also user `x` is not admin and Admin section is not accessible, 
But Admin should be able to start download of URL added by user `x` , that thing is broken.

**Comment by carpecodeum** [comment]
can i work on this?

**Comment by kmehant** [comment]
> can i work on this?

@carpecodeum sure :)
